metrics/lib/features/dashboard/domain/entities/collections/date_time_set.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:collection';
+
+import 'package:collection/collection.dart';
+import 'package:metrics/features/dashboard/domain/entities/collections/date_time_set_entry.dart';
+
+/// Represents the set of the [DateTimeSetEntry]s with the unique date.
```

Comment:
```suggestion
/// Represents the set of the [DateTimeSetEntry]s with elements in set unique by [DateTimeSetEntry.date].
```

maybe something like this? Maybe some additional info what would happen if a new element with the same [DateTimeSetEntry.date] added? Will it be ignored or it will replace the existing element? Or maybe specify that behaviour is same as inr egular Set?